.env.example

```
@@ -7,24 +7,23 @@ DATABASE_URL=postgresql://raven:changeme@postgres:5432/raven?sslmode=disable
-KEYCLOAK_ADMIN=admin
-KEYCLOAK_ADMIN_PASSWORD=changeme
-KC_DB_URL=jdbc:postgresql://postgres:5432/keycloak
-KC_DB_USERNAME=raven
-KC_DB_PASSWORD=changeme
+# ─── Zitadel ─────────────────────────────────────────────────────────────────
+ZITADEL_MASTERKEY=MustBeAtLeast32CharactersLongKey!
+ZITADEL_EXTERNALDOMAIN=localhost
+ZITADEL_EXTERNALPORT=8080
+ZITADEL_EXTERNALSECURE=false
+ZITADEL_DB_PASSWORD=zitadel
+ZITADEL_CLIENT_ID=  # Set after first Zitadel boot (from init output)
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)